src/neo4j/_codec/hydration/_interface/__init__.py

```diff
@@ -18,11 +18,14 @@
 
 import abc
 
+from .._common import DehydrationHooks
+
 
 class HydrationHandlerABC(abc.ABC):
     def __init__(self):
         self.struct_hydration_functions = {}
-        self.dehydration_functions = {}
+        self.dehydration_hooks = DehydrationHooks(exact_types={},
+                                                  subtypes={})
```

Comment:
are you using subtypes somewhere? I've not seen it yet

Reply:
No, I don't. It's presence doesn't hurt performance-wise though (only if there was something in there, could the driver become noticeably slower). I thought I might need it and ended up not needing it in the end. But I still decided to leave it in as it will come in handy should we at some point allow custom dehydration hooks ([internal Trello card](https://trello.com/c/NajZu4P0)).